.gitignore

```diff
@@ -27,3 +27,4 @@ yarn-error.log*
 
 # Sentry Config File
 .sentryclirc
+.env
```

Comment:
can we move it to `# misc`where the rest of `.env.*`are? nothing crucial, just tidier heh